### ISSUE TITLE
Release: can new version of the ingress and dashboard helm charts be released with new features?

### ISSUE THREAD

**Issue by ikatlinsky** [apache/apisix-helm-chart#556]
It would be very helpful if new versions of charts will be released soon, as they contain features that can improve the deployment process of the apisix on Kubernetes.

Gateway itself seems to be released often enough, but not ingress or dashboard.

**Comment by ikatlinsky** [comment]
@tao12345666333 thanks, is there any release guideline documentation? Seems like only versions of the charts needs to be updated, but still..

**Comment by tao12345666333** [comment]
Yes, simply updating the version is enough to release a new version.

**Comment by ikatlinsky** [comment]
thanks, will do that
